crates/subspace-networking/src/constructor.rs

```diff
@@ -221,8 +235,8 @@ pub struct Config<LocalRecordProvider> {
     pub special_target_connections: u32,
     /// Addresses to bootstrap Kademlia network
     pub bootstrap_addresses: Vec<Multiaddr>,
-    /// Kademlia mode. None means "automatic mode".
-    pub kademlia_mode: Option<Mode>,
+    /// Kademlia mode.
```

Comment:
Maybe mention in the comment what default is and why one might want to change it